debug/expressions.html

```diff
@@ -0,0 +1,86 @@
+            }
+        }
+    },
+    layers: [
+        {
+            id: 'circle',
+            type: 'circle',
+            source: 'geojson',
+            paint: {
+                'circle-radius': {
+                    type: 'expression',
+                    expression: [
+                        '*',
+                        [ '^', 2, {ref: 'map', key: 'zoom'} ],
+                        0.125,
+                        [
+                            '+',
+                            20,
+                            {ref: 'feature', key: 'x'},
+                            {ref: 'feature', key: 'y'}
+                        ]
+                    ]
+                },
+                'circle-opacity': {
+                    type: 'expression',
+                    expression: [ '/', [ '+', 10, {ref: 'feature', key: 'x'} ], 30 ]
+                },
+                'circle-color': {
+                    type: 'expression',
+                    expression: [
+                        'concat',
```

Comment:
👍 yep, agreed that having color functions `rgb`, etc. as opposed to using string concatenation. (Ref: https://github.com/mapbox/mapbox-gl-js/pull/4715#issuecomment-302150023)